### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by 4c74356b41**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: articles/site-recovery/site-recovery-active-directory.md [hunk 1]
```diff
@@ -78,19 +78,19 @@ Then [reconfigure the DNS server for the virtual network](../active-directory/ac
 
 ## Test failover considerations
 
-Test failover occurs in a network that's isolated from production network so that there's no impact to production workloads.
+Test failover occurs in a network that's isolated from production network so that there's no impact on production workloads.
```
Comment (4c74356b41) [inline articles/site-recovery/site-recovery-active-directory.md]: I think it's more correct this way

**Comment by PRmerger** [comment]
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @prateek9us, has been notified to review your proposed change.

**Comment by prateek9us** [comment]
@4c74356b41 Thanks for the contribution. 
@PRmerger  This can be merged as-is.

**Comment by prateek9us** [comment]
#sign-off

**Comment by 4c74356b41** [comment]
thanks

**Pull request merged**